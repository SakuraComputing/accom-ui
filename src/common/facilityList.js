import React from 'react';

const facilityList = (data) => {

    if(!data) {
        return null;
    }

    return data.map((node, index) => {
        return <li key ={ node.id } label={ node.label }>
            {node.label}
        </li>
    })

};

export default facilityList;