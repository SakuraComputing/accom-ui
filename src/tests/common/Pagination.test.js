import React from 'react';
import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';
import { Pagination } from '../../common/Pagination';

describe('Pagination Page', () => {
    it('should render pagination correctly',() => {
        const wrapper = shallow(<Pagination />);
        expect(toJSON(wrapper)).toMatchSnapshot();
    });
});


