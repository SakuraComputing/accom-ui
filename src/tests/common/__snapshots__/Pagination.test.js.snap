// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pagination Page should render pagination correctly 1`] = `
<div
  id="pagination-container"
>
  <div
    id="pagination"
  >
    <div
      className="pagination-button"
      onClick={[Function]}
    >
      prev
    </div>
    <ul
      id="pagination-list"
    />
  </div>
</div>
`;
